tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php

```diff
@@ -29,9 +29,13 @@ public function setUp()
     public function testFormTwigTemplate()
     {
         $twig = $this->getContainer()->get('twig');
-        if (class_exists(ContainerAwareRuntimeLoader::class)) {
+        if (class_exists('Symfony\Component\Form\FormRenderer')) {
```

Comment:
I am confused how to catch the renderer now. The deprecation warnings do not really point to the way which is correct now. @xabbuh you had been involed several time here, you got some hints for me? The only public service for me is ` twig` and calling a runtime in twig calls a ContainerLoader which uses `get()` on the container, which causes a waringing of fetching a private service on 3.4.

Reply:
why do you change this line to not use the ::class constant anymore? also, i doubt that this is the class we want to check for - i think this was about finding if the twig bundle is available or not.

Reply:
Just to see which renderer i currently see. Will get ::clas back when it works.